docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -242,3 +316,30 @@ In order to avoid a conflict between multiple Elasticsearch clusters writing the
 - sets the `base_path` to `snapshots/<namespace>-<esName>` when it is not provided, for Azure, GCS and S3 repositories
 - appends `<namespace>-<esName>` to `location` for a FS repository
 - appends `<namespace>-<esName>` to `path` for an HDFS repository
+
+[float]
+[id="{p}-{page_id}-specifics-secret-mounts"]
+== Specifics for secret mounts
+
+In ECK `2.11.0` a new field was introduced to the Elastic Stack configuration policy under the `elasticsearch` field called `secretMounts`.
+This field allows users to specify a user created secret and a mountPath to indicate where this secret should be mounted in the Elasticsearch Pods that are managed by the Elastic Stack configuration policy.
+This field can be used to add additional secrets to the Elasticsearch Pods that may be needed for sensitive files required for configuring Elasticsearch realms.
+The secret should be created by the user is the same namespace as the operator.
+The operator will read this secret and copy it over to the namespace of Elasticsearch so that it can be mounted by the Elasticsearch Pods.
+Example of configuring secret mounts in the Elastic Stack configuration policy:
+
+[source,yaml,subs="attributes,+macros,callouts"]
+----
+secretMounts:
+  - secretName: jwks-secret <1>
+    mountPath: "/usr/share/elasticsearch/config/jwks" <2>
+----
+
+<1> name of the secret created by the user in the operator namespace.
+<2> mount path where the secret must be mounted to inside the Elasticsearch Pod.
+
+[float]
+[id="{p}-{page_id}-configuring-authentication-policies"]
+== Configuring authentication policies using Elastic Stack configuration policy
+
+In ECK `2.11.0` with the introduction of the `config` field for `elasticsearch` and `kibana`, `secretMounts` field for `elasticsearch`, Elastic Stack configuration policy can be used to configure authentication for Elasticsearch clusters. For some examples of the various authentication configurations that can be used refer to <<{p}-auth-config-using-stack-config-policy>>
```

Comment:
Something seems odd after "with the introduction of...." Also referencing ECK versions outside of the release highlights/notes is not something we used to do (outside of a deprecation note `2.11.0` is referenced 6 times in this PR), not sure it really matters but this is something that feels new to me.

```suggestion
Elastic Stack configuration policy can be used to configure authentication for Elasticsearch clusters. Check <<{p}-auth-config-using-stack-config-policy>> for some examples of the various authentication configurations that can be used.
```